packages/jest-types/__typetests__/config.test.ts

```diff
@@ -86,34 +86,34 @@ expectError<Config.InitialOptions>({
 expectAssignable<Config.InitialOptions>({
   fakeTimers: {
     enableGlobally: true,
-    legacyFakeTimers: true,
+    legacyFakeTimers: true as const,
```

Comment:
Here is why I believe that `expectAssignable` does better job.

Just because `Config.InitialOptions` was provided as expectation, the type of `legacyFakeTimers` was inferred as `true`. Hm.. I don’t think that testing tool should infer types based on expectations. Is that testing? How to create difference between `true` and `boolean`? Sounds better if expectation type and expression are isolated and knows nothing about each other.